### PR TITLE
SOC-1880 | Use link-to instead of an action

### PR DESCRIPTION
**Pull request by bkoval**
@kvas-damian @rafalkalinski @Bloodmoon24 
https://wikia-inc.atlassian.net/browse/SOC-1880

### CODE REVIEW

**Comment by Bloodmoon24** [comment]
LGTM

File: front/templates/main/components/discussion-post.hbs [hunk 1]
```diff
@@ -32,7 +32,8 @@
 			}}
 		{{else if model.notFoundError}}
 			{{discussion-not-found-error
-				goToAllDiscussions=(action 'goToAllDiscussions')
+				forumId=model.forumId
+				sortBy=discussionSort.sortBy
```
Comment (kvas-damian) [inline front/templates/main/components/discussion-post.hbs]: You could inject ```discussionSort``` service in ```discussion-not-found-error```.

**Comment by kvas-damian** [comment]
LGTM

**Comment by kvas-damian** [comment]
Linter!

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**